node-rcon.js

```diff
@@ -71,14 +83,25 @@ Rcon.prototype.send = function(data, cmd, id) {
   this._sendSocket(sendBuf);
 };
 
+/**
+ * Sends and establishes a socket connection with the server, either on TCP or UDP connections
```

Comment:
As far as I'm aware, this function doesn't establish a socket connection ever, only writes to existing sockets (and the UDP setup is connectionless). Probably doesn't need function level documentation anyway as it's not intended to be a public function.

Reply:
Alright, fair enough; I'm gonna remove the documentation for this function as you specified to keep up with the consistency of the update